iModelCore/ECPresentation/Source/Shared/ValueHelpers.h

```diff
@@ -27,6 +27,8 @@ struct ValueHelpers : NonCopyableClass
     static BentleyStatus GetEnumPropertyDisplayValue(Utf8StringR displayValue, ECN::ECPropertyCR prop, RapidJsonValueCR jsonValue);
     static BentleyStatus GetEnumPropertyDisplayValue(Utf8StringR displayValue, ECN::ECEnumerationCR enumeration, BeSQLite::DbValue const& dbValue);
 
+    static ECObjectsStatus ParsePrimitiveType(PrimitiveType&, Utf8StringCR);

```

Comment:
We're not in ECObjects library, so I'd rather we use `BentleyStatus` as the return type.

Reply:
Changed